public/Aframe/assets/README.md

```diff
@@ -0,0 +1,16 @@
+# Assets
+
+Include any free or paid for assets in this repo locally.
+
+Note, this is excluded from the git repository. If you notice any changed files showing up in this subfolder, make sure to check the file is meant to be there. If it is, please update the .gitignore in the root directory to include this file, so that it no longer shows as a change in the pull request.
+
+## Syncing Assets
+
+Check the [google drive](https://drive.google.com/drive/folders/1ZMUpKL_NLyeoAz-bUBLWKgDy05wPARhM?usp=sharing) for the latest assets. Please copy into your Aframe/assets folder whenever new assets are added.
+
+## New Assets?
```

Comment:
Sent some links in the discord if you want to add those before merging PR

Reply:
Yup, exactly my plan!